nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/main/java/org/apache/nifi/processors/azure/storage/ListAzureDataLakeStorage.java

```diff
@@ -226,9 +239,11 @@ protected String getPath(ProcessContext context) {
             options.setRecursive(recurseSubdirectories);
 
             Pattern baseDirectoryPattern = Pattern.compile("^" + baseDirectory + "/?");
+            String tempFilePrefix = context.getProperty(TEMP_FILE_PREFIX).evaluateAttributeExpressions().getValue();
```

Comment:
Recommend adding `final` to this variable declaration.

Reply:
final keyword has been added.